tests/manage/pv_services/test_rbd_image_metadata.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+from ocs_ci.framework.pytest_customization.marks import tier2, polarion_id, bugzilla
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@polarion_id("OCS-4465")
+@bugzilla("2099965")
+class TestRbdImageMetadata:
+    """
+    Test if the rbd images have metdata being set
+    """
+
+    def test_rbd_image_metadata(self, pvc_factory, pvc_clone_factory):
+
+        # create a pvc with ceph-rbd
+        pvc_obj = pvc_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            status=constants.STATUS_BOUND,
+            volume_mode="Block",
+        )
+
+        # create a snapshot of the PVC
+        pvc_obj.create_snapshot(
+            snapshot_name=helpers.create_unique_resource_name("test", "snapshot"),
+            wait=True,
+        )
+
+        # create a clone of the PVC
+        pvc_clone_factory(pvc_obj)
+
+        # get all the images
+        ceph_tool_pod = get_ceph_tools_pod()
+        rbd_images = ceph_tool_pod.exec_cmd_on_pod(
+            command=f"rbd ls {constants.DEFAULT_CEPHBLOCKPOOL}"
```

Comment:
The output is the command is line by line. Why the split is with space and not new line? 
csi-vol-0c6ceef9-18d5-11ed-90e4-0a580a800212
csi-vol-0c6fd598-18d5-11ed-90e4-0a580a800212
csi-vol-0cd6d0f8-18d5-11ed-90e4-0a580a800212

Reply:
mmm.. this is the output im seeing
```
csi-snap-050f3dc2-2521-11ed-b025-0a580a830013 csi-vol-0b2c8d64-2521-11ed-b025-0a580a830013 csi-vol-0b2c8d64-2521-11ed-b025-0a580a830013-temp csi-vol-be82f44c-243c-11ed-972b-0a580a810215 csi-vol-cd4f62cf-243c-11ed-972b-0a580a810215 csi-vol-cd584e1b-243c-11ed-972b-0a580a810215 csi-vol-d06b72d4-243c-11ed-972b-0a580a810215 csi-vol-d0754609-243c-11ed-972b-0a580a810215 csi-vol-ff3fda78-2520-11ed-b025-0a580a830013
```

Reply:
It fails with newline.

Reply:
Maybe this is because `exec_cmd_on_pod` is calling `exec_oc_cmd` which is returning yaml output which is not relevant here. You can try to send the command with arg `out_yaml_format=False` or use instead https://github.com/red-hat-storage/ocs-ci/blob/7500cbf39bcd70eb9954807dc860f835efdd1e34/ocs_ci/ocs/resources/pod.py#L215 which is for regular commands in bash.

Reply:
It comes with extra space in the list when I split with newline, ``` ['csi-snap-8d01797a-29d3-11ed-aaa5-0a580a800214', 'csi-vol-86b3d0e6-29d3-11ed-aaa5-0a580a800214', 'csi-vol-942c651a-29d3-11ed-aaa5-0a580a800214', 'csi-vol-942c651a-29d3-11ed-aaa5-0a580a800214-temp', 'csi-vol-b647da9f-29c6-11ed-aaa5-0a580a800214', 'csi-vol-c48fe851-29c6-11ed-aaa5-0a580a800214', 'csi-vol-c49f54b6-29c6-11ed-aaa5-0a580a800214', 'csi-vol-c808aeb1-29c6-11ed-aaa5-0a580a800214', 'csi-vol-c8107230-29c6-11ed-aaa5-0a580a800214', ''] ```
I feel, Its better the way it is now, instead of adding extra lines of code

Reply:
If it works, no problem. In which method did it comes with space? `out_yaml_format=False` or `exec_sh_cmd_on_pod`. Anyway I'm resolving the issue.